tests/unit/test_cost.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime, timedelta
```

Comment:
Double check how the actions are executing the tests, and do the following:
* Configure the tests job as a matrix job
* One job will execute the api tests the other the unit tests as defined here
* Configure the codecov flag to classify this test as unit.